packages/kvstore/lib/FeatureFlags.ts

```diff
@@ -1,37 +1,15 @@
-import { getLogger } from '@nangohq/utils';
 import type { KVStore } from './KVStore.js';
 
-const logger = getLogger('FeatureFlags');
-
 export class FeatureFlags {
-    kvstore: KVStore | undefined;
-
-    constructor(kvstore: KVStore | undefined) {
-        if (!kvstore) {
-            logger.error('Feature flags not enabled');
-        }
+    kvstore: KVStore;
 
+    constructor(kvstore: KVStore) {
         this.kvstore = kvstore;
     }
 
-    async isEnabled({
-        key,
-        distinctId,
-        fallback,
-        isExcludingFlag = false
-    }: {
-        key: string;
-        distinctId: string;
-        fallback: boolean;
-        isExcludingFlag?: boolean;
-    }): Promise<boolean> {
-        if (!this.kvstore) {
-            return fallback;
-        }
-
+    async isSet(key: string, { distinctId = 'global', fallback = false }: { distinctId?: string; fallback?: boolean } = {}): Promise<boolean> {
         try {
-            const exists = await this.kvstore.exists(`flag:${key}:${distinctId}`);
-            return isExcludingFlag ? !exists : exists;
+            return await this.kvstore.exists(`flag:${key}:${distinctId}`);
```

Comment:
why isExcludingFlag logic being removed?

Reply:
I found it really hard to understand vs just writing a flag name that makes sense. 
e.g: "webhooks fallback false with excluding", means disable this feature when it's set, whereas just setting a feature flag `disable-webhook` is straight forward. Let me know

Reply:
the idea is to always think in term of flag being enabled.
Without it you then have something like `ff.isEnabled('disable-webhook')` which is also not straightforward to read (ie: _is the flag to disable webhook enabled?_) 
I don't feel strongly about it so feel free to remove it

Reply:
I have renamed it to isSet to avoid this confusion `ff.isSet('disable-webhook')` is "clear" = no double negative